em-app/src/lib.rs

```diff
@@ -138,6 +139,14 @@ pub fn request_issue_certificate(url: &str, csr_pem: String) -> Result<models::I
     client.issue_certificate(request).map_err(|e| Error::NodeAgentClient(Box::new(e)))
 }
 
+/// Computes application configuration hash from raw string
+pub fn compute_app_config_hash(app_config: &str) -> Result<Blob> {
+    let mut digest = vec![0; 32];
+    hash::Md::hash(hash::Type::Sha256, app_config.as_bytes(), &mut digest).map_err(|e| Error::TargetReportHash(Box::new(e)))?;
```

Comment:
Should we also pass hash type to the function in addition to the app_config string? Do we always use sha256 hashes only?

Reply:
changed